.eslintrc.js

```diff
@@ -0,0 +1,98 @@
+'use strict';
+
+const restrictedGlobals = require('confusing-browser-globals');
+
+const OFF = 0;
+const ERROR = 2;
+
+module.exports = {
+  extends: ['fbjs', 'prettier'],
+
+  // Stop ESLint from looking for a configuration file in parent folders
+  root: true,
+
+  plugins: ['jest', 'no-for-of-loops', 'react'],
+
+  parser: 'babel-eslint',
+  parserOptions: {
+    ecmaVersion: 8,
+    sourceType: 'module',
+    ecmaFeatures: {
+      experimentalObjectRestSpread: true,
+    },
+  },
+
+  // We're stricter than the default config, mostly. We'll override a few rules
+  // and then enable some React specific ones.
+  rules: {
+    'accessor-pairs': OFF,
+    'brace-style': [ERROR, '1tbs'],
+    'consistent-return': OFF,
+    'dot-location': [ERROR, 'property'],
+    // We use console['error']() as a signal to not transform it:
+    'dot-notation': [ERROR, {allowPattern: '^(error|warn)$'}],
+    'eol-last': ERROR,
+    eqeqeq: [ERROR, 'allow-null'],
+    indent: OFF,
+    'jsx-quotes': [ERROR, 'prefer-double'],
+    'keyword-spacing': [ERROR, {after: true, before: true}],
+    'no-bitwise': OFF,
+    'no-inner-declarations': [ERROR, 'functions'],
+    'no-multi-spaces': ERROR,
+    'no-restricted-globals': [ERROR].concat(restrictedGlobals),
+    'no-restricted-syntax': [ERROR, 'WithStatement'],
+    'no-shadow': ERROR,
+    'no-unused-expressions': ERROR,
+    'no-unused-vars': [ERROR, {args: 'none'}],
+    'no-use-before-define': OFF,
+    'no-useless-concat': OFF,
+    quotes: [ERROR, 'single', {avoidEscape: true, allowTemplateLiterals: true}],
+    'space-before-blocks': ERROR,
+    'space-before-function-paren': OFF,
+    'valid-typeof': [ERROR, {requireStringLiterals: true}],
+
+    'no-var': ERROR,
+    'prefer-const': ERROR,
+    strict: OFF,
+
+    // Enforced by Prettier
+    // TODO: Prettier doesn't handle long strings or long comments. Not a big
+    // deal. But I turned it off because loading the plugin causes some obscure
+    // syntax error and it didn't seem worth investigating.
+    'max-len': OFF,
```

Comment:
I believe this was what might be causing the error with my draft #18 , apparently `fbjs` plugin causes a syntax error at `max-len` and doesn't work. 
Maybe something worth reporting to fbjs? But they explicitly mention they don't maintain it much and don't want people to use it so I guess not.

Reply:
Ah interesting. Yeah sounds like they wont want bug reports for this.

By the way just for clarity, this (and the comment) was just copied from React’s config, I didn’t actually look into it hahaha.